src/main/ml-modules/services/geoQueryService.sjs

```diff
@@ -974,6 +960,25 @@ function getPlanForDataSource(dataSource) {
   }
 }
 
+function updateWhereWithObjectIds(query, whereQuery) {
+  const ids = parseObjectIds(query);
+  if (ids) {
+    // this assumes we are querying against the OBJECTID field as a number
+    // should use a range index if we have one
+    //const idsQuery = cts.jsonPropertyValueQuery("OBJECTID", ids.map(Number));
+    xdmp.trace("GDS-DEBUG", "getting ids: " + ids);
+
+    //boundingQueries.push(idsQuery);
```

Comment:
Let's smoke these two commented lines, they look like zero value. As for the 3 lines above it - I'm thinking smoke those as well. We have a ticket already for not assuming that "OBJECTID" is the correct field, and this code will be easy to find now.